sopel/config/core_section.py

```diff
@@ -302,45 +390,108 @@ class CoreSection(StaticSection):
 
     :default: ``UTC``
 
+    .. highlight:: ini
+
     Used when plugins format times with :func:`sopel.tools.time.format_time`.
+
+    Example with the British time::
```

Comment:
```suggestion
    For example, to make Sopel fall back on British time::
```

(idk, still kinda messy)